modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm

```diff
@@ -0,0 +1,94 @@
+/obj/item/clothing/under/dress
+	body_parts_covered = CHEST|GROIN	//For reference
```

Comment:
female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY (see this PR for what it does TL;DR: https://github.com/Skyrat-SS13/Skyrat-tg/pull/14927)

Reply:
Added it 👍